org-roam-capture.el

```diff
@@ -80,7 +80,13 @@ note with the given `ref'.")
   "Keywords used in `org-roam-capture-templates' specific to Org-roam.")
 
 (defvar org-roam-capture-templates
-  '(("d" "default" plain (function org-roam-capture--get-point)
+  '(("a" "Automatic" plain (function org-roam--capture-get-point)
```

Comment:
the automatic template might want to be moved out of `org-roam-capture-templates`, since it's unlikely the user wants to use the automatic template in a normal insert/find-file command.

Reply:
Yes I agree. That’s why I think it would be good to just let a user define their own template key for auto-creation and throw errors if they don’t have one set when calling the auto-create functions. 

It may also be possible to move this default auto-template to its own var and use it for auto-creation unless a user sets their own key. 

Either is fine with me!

Reply:
See the new commits